packages/server-wallet/src/__test__/sync-objectives.test.ts

```diff
@@ -0,0 +1,152 @@
+import {CreateChannelParams} from '@statechannels/client-api-schema';
```

Comment:
nit: this file ought to be called something a bit different -- otherwise, it's not obvious why it's separate from the test file included below.

I think there's a set of tests which play two wallets against each other. Perhaps they should be grouped under `__test__/with-peers/...` or something like that.

Reply:
I agree! I think that could be done under this issue: https://github.com/statechannels/statechannels/issues/3279